packages/hardhat-zksync-verify/src/zkscan/ZkScanService.ts

```diff
@@ -0,0 +1,53 @@
+import axios from 'axios';
```

Comment:
Rename to ZkSyncBlockExplorerService pls -  just to avoid confusion given zkScan will still exist for zkSync v1